controllers/admin/AdminModulesController.php

```diff
@@ -1395,10 +1395,32 @@ public function initModal()
     {
         parent::initModal();
 
+        $module = Module::getInstanceByName(Tools::getValue('configure'));
+        $languages = Language::getLanguages(false);
+        $isNewTranslateSystem = $module->isUsingNewTranslationsSystem();
+        $translateLinks = array();
+        foreach ($languages as $lang) {
+            if ($isNewTranslateSystem) {
+                $translateLinks[$lang['iso_code']] = Context::getContext()->link->getAdminLink('AdminTranslationSf', true, array(
```

Comment:
You could extract `Context::getContext()->link` into a variable in order to avoid repeated calls to `Context::getContext()`